homeassistant/core.py

```diff
@@ -1219,7 +1219,16 @@ async def async_call(
             raise ServiceNotFound(domain, service) from None
 
         if handler.schema:
-            processed_data = handler.schema(service_data)
+            try:
+                processed_data = handler.schema(service_data)
+            except vol.Invalid:
+                _LOGGER.debug(
```

Comment:
Should we make this `_LOGGER.error` so they don't have to turn on debug mode to get context about the error line that is going to be right next to it in the log?

Reply:
It is not error of a component, but on the other hand if user has invalid configuration for any component, it would log it as an error.
a `_LOGGER.warning` maybe?

Reply:
`_LOGGER.warning` seems fine as well since we expect the user will see it.

Reply:
I thought about this before deciding on debug. First, it's not _another_ error; it's just additional details about _the_ error. Logging another error might confuse people leading them to think there were two different errors. So I though of extending the exception object with additional data (in its `args` field) and re-raising it, but that was more complicated than I expected. Second, there are plenty of examples of needing to enable debug when one wants to, well, debug. Third, I didn't want to change the current behavior, but rather just add to it.

At least, that was my reasoning. Also, I would definitely not go with warning, since it's not a warning, it's part of an error.

Whatever y'all decide I guess I'm ok with, although I still think debug is the right choice.

Reply:
I don't have strong feelings on this (thus the approval) so I'd say do what you think is best.

Reply:
This is a good first step, we can add more fancy things in the future.